future_challenges_for_RSEng.tex

```diff
@@ -169,6 +169,25 @@ \subsection{How have academic Institutions transformed by then? ( Point 4 in the
   - Encourage knowledge exchange to find models that suit different organizational constraints.
 
 \subsection{Ethics and social consideration}
+The ethics and values of the Research Software Engineer are grounded in their practice.
+They commit to the health, safety and welfare of the public and act in the interest of society,
+their employer and their clients \cite{Goth2023}.
+Sometimes these values are in direct conflict with the aims of their institution. 
+For example, their PI might insist on quickly developing some software without adhering to current
+best practices. 
+Another example might be that they are developing code that can also be used for military purposes,
```

Comment:
I would avoid this example to avoid the whole discussion about military use from here.